packages/react/src/components/table/table-row.tsx

```diff
@@ -114,7 +114,7 @@ const StyledTableRow = styled.tr<StyledTableRowProps>`
         }
 
         td:first-child::after {
-            border-left: 1px solid ${theme.notifications['alert-2.1']};
+            border-left: 1px solid ${theme.component['table-row-shadow-color']};
```

Comment:
Ici je suis pas certain, mais de quoi du genre:
```suggestion
            border-left: 1px solid ${theme.component['table-first-row-border-left-color']};
```

Reply:
Meme commentaire, c'est la bordure d'un row avec erreur.